samples/server/petstore/java-play-framework-async/app/openapitools/SecurityAPIUtils.java

```diff
@@ -0,0 +1,165 @@
+package openapitools;
```

Comment:
this sample is not using security at all. Does it compile?

These lines will not return anything: 

```
clientId = configuration.getString("oauth.clientId");
clientSecret = configuration.getString("oauth.clientSecret");
```

Reply:
ok, I will push a new version that will address compilation and potential runtime throw issues.